recipe/meta.yaml

```
@@ -30,20 +30,20 @@ outputs:
         - setuptools
         - wheel
         - packaging
-        - param 1.9.0
+        - param >=1.9.0,<3.0
         - pyct 0.5.0
```

```suggestion
        - pyct
```
We can drop host pinnings for most of the python packages